android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -288,6 +289,13 @@ public void processProperties(KrollDict d)
 			}
 		}
 
+		if (d.containsKey(TiC.PROPERTY_HTML)) {
+			Spanned text = Html.fromHtml(TiConvert.toString(d, TiC.PROPERTY_HTML));
+			if (text != null) {
+				tv.setText(text, TextView.BufferType.SPANNABLE);
```

Comment:
@m1ga I believe this should be `TextView.BufferType.EDITABLE` to allow editing as well. `SPANNABLE` is only for formatting with styled text.

Reply:
It should actually not be editable, same as for iOS. If we use HTML + editing, that would be a RTE like https://github.com/hansemannn/titanium-rich-text-editor

Reply:
it's still an editable text field and you can change the text but it doesn't allow you to change the format of course. 

I don't know what the iOS version does :) This PR will just allow you to set a HTML text in a normal TextArea:
![Screenshot_20250416-141650](https://github.com/user-attachments/assets/58fd00cc-d9ef-400c-9792-cd88378d52e7)
same way the label does it, but here just as an editable field

Reply:
The docs didn't mention that this property makes the TextArea uneditable. We should actually check for the existing `editable` property to decide the `BufferType` value for flexibility.

Reply:
> The docs didn't mention that this property makes the TextArea uneditable. We should actually check for the existing `editable` property to decide the `BufferType` value for flexibility.

https://developer.android.com/reference/android/widget/TextView.BufferType#EDITABLE
is not really helpful :smile: No explanation.

According to https://stackoverflow.com/questions/65642029/the-effect-of-buffertype-normal-and-buffertype-spannable-for-textview or https://stackoverflow.com/a/41002135/5193915 it doesn't really matter in this case and I think SPANNBLE might even be better as I'm adding a spanned element in there and not a plain text.